schema/crdb/collapse_lldp_settings/up1.sql

```diff
@@ -0,0 +1,4 @@
+/*
+ * The old lldp_service_config_id is being replaced with lldp_link_config_id.
+ */
+ALTER TABLE omicron.public.switch_port_settings_link_config DROP COLUMN IF EXISTS lldp_service_config_id;
```

Comment:
This column and the tables below can be dropped without losing any data, as no prior release has had any way to populate any of them.  They were added as placeholders/stakes-in-the-ground last year.